pkg/api/v1/resource/helpers.go

```diff
@@ -65,37 +68,75 @@ func PodRequestsAndLimits(pod *v1.Pod) (reqs, limits v1.ResourceList) {
 		maxResourceList(reqs, container.Resources.Requests)
 		maxResourceList(limits, container.Resources.Limits)
 	}
+
+	// if PodOverhead feature is supported, add overhead for running a pod
+	// to the non-zero sum of container requests/limits:
+	if pod.Spec.Overhead != nil && utilfeature.DefaultFeatureGate.Enabled(features.PodOverhead) {
```

Comment:
There were concerns, which surprised me.  We ended up keeping it as is, since it is first checked against the Overhead being a part of the pod.Spec, anyway.  It'd be abnormal to hit this check and have Overhead set and the feature gate not set.